baldrick/__init__.py

```diff
@@ -3,7 +3,7 @@
 from flask import Flask
 from werkzeug.contrib.fixers import ProxyFix
 
-from baldrick.blueprints import github, circleci
+from baldrick.blueprints import github as github_blueprint, circleci
```

Comment:
@Cadair - the former name caused issues because it caused ``baldrick.github`` to refer to the blueprint not the sub-package. Shall we just rename the actual blueprints to github_blueprint and circleci_blueprint?